cpu/stm32/stmclk/stmclk_u5.c

```diff
@@ -267,7 +242,7 @@ void stmclk_init_sysclk(void)
         while (!(RCC->CR & RCC_CR_HSERDY)) {}
     }
 
-    /* Enable the MSI clock only when it's used */
+    /* Enable the MSIS clock only when it's used */
```

Comment:
Is this a typo? Or should MSI be replaced by MSIS consistently?

Reply:
:thinking: good question. It is used by intention here. There is one clock generator MSI which can generate the two different output clocks MSIS and MSIK that have to be enabled separately. In fact, only the MSIS output is configured here but the clock generator is the MSI.